src/System.Net.Sockets/tests/FunctionalTests/SocketOptionNameTest.cs

```diff
@@ -86,6 +87,11 @@ public void MulticastOption_CreateSocketSetGetOption_GroupAndInterfaceIndex_SetS
         [Fact]
         public async Task MulticastInterface_Set_AnyInterface_Succeeds()
         {
+            if (PlatformDetection.IsFedora)
```

Comment:
Should we use ActiveIssue and specify the test platform to be disabled?

Reply:
I don't believe ActiveIssue currently supports this granular level of platform discernment.  I could use [ConditionalFact], but there's currently no IsNotFedora on PlatformDetection.  Since that would effectively be the same thing, I just went with this.

Reply:
cc: @safern @danmosemsft 
I thought we recently added the ability to have certain Unix variants' (distros) tests disabled via ActiveIssue. Perhaps that is not implemented yet?

Reply:
@safern added support to ActiveIssue for TargetFrameworkMonikers.  I don't see any support in buildtools for ActiveIssue taking a distro.

Reply:
No, the ability of ActiveIssue is only for specific TFMs. Might be worth opening an issue to consider adding that ability as might be useful.

Reply:
Btw AFAIK it wouldn't be that simple but I could investigate if it is possible.

Reply:
>Btw AFAIK it wouldn't be that simple 

I don't understand why it is hard.  We already have `PlatformDetection.IsFedora`.  Why can't that logic be used with ActiveIssue?

Reply:
> Why can't that logic be used with ActiveIssue?

Is not that simple, is about the flags that we pass to xunit and that we add to the tests, so when we create the RunTests.cmd script we need to know the specific OS from the msbuild target and IDK if we know that, from as far as I know we only know if we target Unix/Linux, but not the specific OS. See (https://github.com/dotnet/corefx/blob/master/tools-local/osgroups.props)